### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#119] [Comments]
https://hutusi.com/about/ 

 任抛星汉归园圃，留取乾坤盛酒囊。

**Comment by ZhangJun1015** [comment]
不错，加油

**Reply by hutusi** [reply]
加油😀

**Comment by acegao9527** [comment]
挺好，有机会上海当面交流

**Reply by hutusi** [reply]
好呀🤝

**Comment by seyeeL** [comment]
自认为是终身学习者，只是经常忘了学习。最主要的原因是三分钟热度，不过后来我知道了一个词叫 Yak Shaving, 于是我觉得这也没什么不好。如果缺点改正不了，那就把它当成是优点，会好很多。
---
这一段哈哈哈哈同本人

**Reply by hutusi** [reply]
懒是人类进步的动力😀

**Comment by Moosphan** [comment]
How to display the count of blog views?

**Reply by hutusi** [reply]
用的不蒜子统计：https://busuanzi.ibruce.info/

**Comment by yuesh1** [comment]
真是一个有趣的博主，关注一下，期待产出更多有价值的文章！

**Reply by hutusi** [reply]
谢谢关注：）我也希望自己写出更多有价值的文章~~

**Comment by Caterpillar945** [comment]
牛

**Reply by hutusi** [reply]
加油，hacker girl 👧

**Comment by wangyunzi** [comment]
博客简洁，内容也有很有趣🥰🥰🥰

**Reply by hutusi** [reply]
谢谢，你的博客很赞，看文章还是学生吧，厉害👍 

加了友链，期待互访😄

**Comment by wangyunzi** [comment]
新年快乐！希望博主:
🏮 2023🏮
㊗️ 心想事成 🧧 恭喜发财 ㊗️
🧨 ʜᴀᴘᴘʏ new ʏᴇᴀя 🧨
♥新年快乐 阖家幸福OωO
🎇 愿新的一年
🧨有趣有盼 福气不让 🧧
❤️平安喜乐 大展宏🐰

**Reply by hutusi** [reply]
谢谢：）新年快乐！

**Comment by moreoronce** [comment]
博主你好，请问 giscus在墙内的访问是正常的吗？

**Reply by hutusi** [reply]
可以访问的

**Reply by DeppWang** [reply]
赞

**Comment by edonyzpc** [comment]
博主你好，看到你的 about 页面中的「读书写字~~和编码~~为乐」，我印象中你是在华为，现在是不是不做 coding 相关的工作了，自己比较好奇的是关于 coding 后续发展考虑的问题。

**Reply by hutusi** [reply]
可以加我微信聊哈~~

**Comment by XINBOY722** [comment]
膜拜大佬😭😭😭，我把您的博客添加到我的友链里了，大佬博客内容都好高质量

**Reply by hutusi** [reply]
谢谢，共勉 🤝

**Comment by HiraethEcho** [comment]
好棒的博客！两年前开始随便写自己的独立博客，逐渐发展到现在，似乎和您的博客很多理念都一致，十分惊喜。正在从这里学习博客写作，希望可以多多交流。感谢！

**Reply by hutusi** [reply]
谢谢👏🏻握手🤝一起加油😀
